site/content/docs/5.2/utilities/text.md

```diff
@@ -40,7 +40,7 @@ Wrap text with a `.text-wrap` class.
 Prevent text from wrapping with a `.text-nowrap` class.
 
 {{< example >}}
-<div class="text-nowrap bg-light border border-dark" style="width: 8rem;">
+<div class="text-nowrap bg-body-secondary border border-dark" style="width: 8rem;">
```

Comment:
**[Bootstrap Side]** Can't we try to have consistents example designs (compare to https://deploy-preview-1757--boosted.netlify.app/docs/5.2/utilities/overflow/)